Gems/Atom/RPI/Code/Source/RPI.Builders/Model/MaterialAssetBuilderComponent.cpp

```diff
@@ -78,7 +78,9 @@ namespace AZ
             AZStd::string materialTypePath;
             RPI::MaterialConverterBus::BroadcastResult(materialTypePath, &RPI::MaterialConverterBus::Events::GetMaterialTypePath);
 
-            if (conversionEnabled && !materialTypePath.empty())
+            bool enableMaterialPropertyNames = false;
+            AZ::Interface<AZ::IConsole>::Get()->GetCvarValue("r_enableMaterialPropertyNames", enableMaterialPropertyNames);
```

Comment:
I would prefer this uses the registry system via MaterialConverterSettings and MaterialConverterBus for consistency. We shouldn't have two different ways to configure the material conversion process.

Reply:
The setting should be included in MaterialAssetDependenciesComponent::AddFingerprintInfo so that the AP will automatically reprocess the model files when the user changes the setting.

Reply:
What do you think about calling the setting "include material property names" instead of "enable"?

Reply:
"Include material property names" does sound more appropriate.